packages/ui/src/ValueView/index.tsx

```diff
@@ -63,16 +87,30 @@ export const ValueViewComponent = <SelectedContext extends Context = 'default'>(
       if={!context || context === 'default'}
       then={children => (
         <Pill priority={priority}>
-          <div className={cn('-ml-2', density === 'sparse' ? 'mt-0 mb-0' : '-mt-1 -mb-1')}>
+          <div
+            className={cn(!hideIcon && '-ml-2', density === 'sparse' ? 'mt-0 mb-0' : '-mt-1 -mb-1')}
+          >
             {children}
           </div>
         </Pill>
       )}
+      else={children => (
+        <span
+          className={cn(
+            density === 'sparse' ? technical : detailTechnical,
+            priority === 'primary' ? 'text-text-primary' : 'text-secondary-dark',
+          )}
+        >
+          {children}
+        </span>
```

Comment:
Needed to have this else branch as the Pill was giving the font/color in the other branch and this one wasn't getting any.